manifests/jail.pp

```diff
@@ -27,32 +31,34 @@
   Optional[String] $config_file_notify = $fail2ban::config_file_notify,
   Optional[String] $config_file_require = $fail2ban::config_file_require,
 ) {
-  # Jail filter creation
-  file { "custom_filter_${name}":
-    ensure  => file,
-    path    => "${config_dir_filter_path}/${name}.conf",
-    content => epp('fail2ban/common/custom_filter.conf.epp',
-      {
-        findtime                 => $findtime,
-        filter_includes          => $filter_includes,
-        filter_additional_config => $filter_additional_config,
-        filter_failregex         => $filter_failregex,
-        filter_ignoreregex       => $filter_ignoreregex,
-        filter_maxlines          => $filter_maxlines,
-        filter_datepattern       => $filter_datepattern,
-        journalmatch             => $journalmatch,
-      }
-    ),
-    owner   => $config_file_owner,
-    group   => $config_file_group,
-    mode    => $config_file_mode,
-    notify  => $config_file_notify,
-    require => $config_file_require,
+  if $filter_manage {
```

Comment:
Conditionally manage filter file EG, OS-supplied filter but custom jail for setting tunables